### PR TITLE
OPENEUROPA-1109: Setup project repository

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: site-a/web/sites/default/default.settings.php [hunk 1]
```diff
@@ -771,6 +771,10 @@
 
 
 
+
+
+
+
```
Comment (ademarco) [inline site-a/web/sites/default/default.settings.php]: We should remove this. It's an issue with `nuvoleweb/robo-config` which has been addressed here: https://github.com/nuvoleweb/robo-config/issues/9

Updating composer.lock on all sites will solve the issue.

Comment (ademarco) [reply site-a/web/sites/default/default.settings.php]: Robo Config 0.2.2 has been released, we will need to update sites composer.locks.

File: site-a/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# OpenEuropa template for Drupal projects
```
Comment (voidtek) [inline site-a/README.md]: This README.md could be removed.

File: site-b/README.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
+# OpenEuropa template for Drupal projects
```
Comment (voidtek) [inline site-b/README.md]: This README.md could be removed.

File: site-c/README.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
+# OpenEuropa template for Drupal projects
```
Comment (voidtek) [inline site-c/README.md]: This README.md could be removed.

File: site-c/grumphp.yml.dist [hunk 5]
```diff
@@ -0,0 +1,22 @@
+imports:
```
Comment (voidtek) [inline site-c/grumphp.yml.dist]: This grumphp.yml.dist could be removed.

File: site-b/grumphp.yml.dist [hunk 6]
```diff
@@ -0,0 +1,22 @@
+imports:
```
Comment (voidtek) [inline site-b/grumphp.yml.dist]: this grumphp.yml.dist file could be removed.

File: site-a/grumphp.yml.dist [hunk 7]
```diff
@@ -0,0 +1,22 @@
+imports:
```
Comment (voidtek) [inline site-a/grumphp.yml.dist]: this grumphp.yml.dist file could be removed.

File: docker-compose.yml.dist [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+#    ports:
+#      - 3306:3306
+
+  selenium:
+    image: selenium/standalone-chrome-debug:3.11
+    environment:
+    - DISPLAY=:99
+    - SE_OPTS=-debug
+    ports:
+    - "5900:5900"
```
[the remark this answers is not in the document]
Comment (ademarco) [reply docker-compose.yml.dist]: Well I''ve been consistent by adding them everywhere! :smile:

File: docker-compose.yml.dist [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    image: openeuropa/pcas-server
+    expose:
+    - 8001
+
+  sparql:
+    image: tenforce/virtuoso:1.3.1-virtuoso7.2.4
+    environment:
+      SPARQL_UPDATE: "true"
+    ports:
+    - "8890:8890"
```
[the remark this answers is not in the document]
Comment (ademarco) [reply docker-compose.yml.dist]: See above comment  :smile:

File: docker-compose.yml.dist (hunk 8 above)
[the remark this answers is not in the document]
Comment (voidtek) [reply docker-compose.yml.dist]: :+1:

File: docker-compose.yml.dist (hunk 9 above)
[the remark this answers is not in the document]
Comment (voidtek) [reply docker-compose.yml.dist]: :+1:

**Pull request merged**